mne/io/eeglab/eeglab.py

```diff
@@ -104,11 +105,23 @@ def _eeg_has_montage_information(eeg):
     return has_pos
 
 
-def _get_eeg_montage_information(eeg, get_pos):
-
-    pos_ch_names, ch_names, pos = list(), list(), list()
+def _get_montage_information(eeg, get_pos):
+    """Get channel name, type and montage information from ['chanlocs']."""
+    ch_names, ch_types, pos_ch_names, pos = list(), list(), list(), list()
     for chanloc in eeg.chanlocs:
+        # channel name
         ch_names.append(chanloc['labels'])
+
+        # channel type
+        ch_type = 'eeg'
+        try_type = chanloc.get('type', None)
+        if isinstance(try_type, str):
+            try_type = try_type.strip().lower()
+            if try_type in _PICK_TYPES_KEYS:
+                ch_type = try_type
```

Comment:
Actually thinking about this functionality now, maybe it's a good idea to warn people if we encounter unknown types? For example something like this could help
```
        unknown_types = set()
        ...
            if try_type in _PICK_TYPES_KEYS:
                ch_type = try_type
            else:
                unknown_types.add(ch_type)
        if len(unknown_types):
            warn(f'Unknown types found, setting as type EEG: {sorted(unknown_types)}')
```

Reply:
... or maybe better, keep a list of each channel name that was a given unknown type, then warn with
```
Unknown types found setting as type EEG:
'something': ['somechan1', 'somechan2']
'someelse': ['someelsechan1']
```

Reply:
True, I wanted to leave untouched the default behavior to set type as `'eeg'`, but a warning would be nice! I'll add it.

Reply:
Barely saw your second reply, but done.